systemtest/src/test/java/io/strimzi/systemtest/BaseST.java

```diff
@@ -165,36 +197,58 @@ protected void teardownEnvForOperator() {
+        recreateTestEnv(coNamespace, Collections.singletonList(coNamespace), operationTimeout, reconciliationInterval);
+    }
+
+    /**
+     * Recreate namespace and CO after test failure
+     * @param coNamespace namespace where CO will be deployed to
+     * @param bindingsNamespaces array of namespaces where Bindings should be deployed to.
+     */
+    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces) throws Exception {
+        recreateTestEnv(coNamespace, bindingsNamespaces, Constants.CO_OPERATION_TIMEOUT_DEFAULT, Constants.RECONCILIATION_INTERVAL);
+    }
+
     /**
      * Recreate namespace and CO after test failure
      * @param coNamespace namespace where CO will be deployed to
      * @param bindingsNamespaces array of namespaces where Bindings should be deployed to.
+     * @param operationTimeout timeout for CO operations
      */
-    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces) throws InterruptedException {
-        recreateTestEnv(coNamespace, bindingsNamespaces, Constants.CO_OPERATION_TIMEOUT_DEFAULT);
+    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces, long operationTimeout) throws Exception {
+        recreateTestEnv(coNamespace, bindingsNamespaces, operationTimeout, Constants.RECONCILIATION_INTERVAL);
     }
 
     /**
      * Recreate namespace and CO after test failure
      * @param coNamespace namespace where CO will be deployed to
      * @param bindingsNamespaces array of namespaces where Bindings should be deployed to.
      * @param operationTimeout timeout for CO operations
+     * @param reconciliationInterval CO reconciliation interval
      */
-    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces, long operationTimeout) {
+    protected void recreateTestEnv(String coNamespace, List<String> bindingsNamespaces, long operationTimeout, long reconciliationInterval) throws Exception {
         ResourceManager.deleteMethodResources();
         ResourceManager.deleteClassResources();
 
         KubeClusterResource.getInstance().deleteClusterOperatorInstallFiles();
         KubeClusterResource.getInstance().deleteNamespaces();
 
-        KubeClusterResource.getInstance().createNamespaces(coNamespace, bindingsNamespaces);
-        KubeClusterResource.getInstance().applyClusterOperatorInstallFiles();
-
-        ResourceManager.setClassResources();
-
-        applyRoleBindings(coNamespace, bindingsNamespaces);
-        // 050-Deployment
-        KubernetesResource.clusterOperator(coNamespace, operationTimeout).done();
+//        KubeClusterResource.getInstance().createNamespaces(coNamespace, bindingsNamespaces);
+//        KubeClusterResource.getInstance().applyClusterOperatorInstallFiles();
+//
+//        ResourceManager.setClassResources();
+//
+//        applyRoleBindings(coNamespace, bindingsNamespaces);
+//        // 050-Deployment
```

Comment:
Did you mean to leave this commented out?

Reply:
Good catch, I forget to remove it :)